src/lib/components/Tooltip.svelte

```diff
@@ -56,35 +53,54 @@
     // transform should be added to the existing one.
     tooltipTransformX += x;
     tooltipTransformY += y;
-  });
+  };
+
+  const onMouseEnter = () => {
+    setPosition();
+    targetIsHovered = true;
+  };
 
-  $: innerWidth, tooltipComponent, target, text, setPosition();
+  const onMouseLeave = () => {
+    targetIsHovered = false;
+  };
+
+  onMount(async () => {
+    // Move tooltip to the body to avoid it being cut off by overflow: hidden.
```

Comment:
It's not possible for DOM elements to have multiple parents. If you append an existing element to a new parent, it will automatically be removed from its previous parent.
So this does indeed move the element.